plugins/org.jboss.tools.websockets.core/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,20 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %PLUGIN_NAME
+Bundle-SymbolicName: org.jboss.tools.websockets.core;singleton:=true
+Bundle-Version: 1.8.0.qualifier
+Bundle-Localization: plugin
+Bundle-Activator: org.jboss.tools.websockets.core.WebsocketsCorePlugin
+Require-Bundle: org.eclipse.ui;bundle-version="3.7.0",
+ org.eclipse.core.runtime;bundle-version="3.7.0",
+ org.eclipse.core.resources;bundle-version="3.7.0",
+ org.eclipse.jdt.core;bundle-version="3.7.0",
+ org.eclipse.jdt.launching;bundle-version="3.6.1",
+ org.eclipse.jdt.launching;bundle-version="3.6.1",
+ org.eclipse.jdt.launching;bundle-version="3.6.1",
+ org.jboss.tools.foundation.ui,
```

Comment:
UI dependencies (org.eclipse.ui and org.jboss.tools.foundation.ui) should not be required in Core bundle, IMO.  
Plus, I don't see where in the core bundle you need the org.eclipse.jdt.core and org.eclipse.jdt.launching dependencies